ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -77,6 +110,35 @@ function BlockaidBannerAlert({ txData, ...props }) {
   } = securityAlertResponse;
 
   if (resultType === BlockaidResultType.Benign) {
+    if (isLoadingDisplayed) {
+      return (
+        <Box
+          alignItems={AlignItems.center}
+          borderColor={BorderColor.borderDefault}
+          borderRadius={BorderRadius.SM}
+          padding={2}
+          display={Display.Flex}
+        >
+          <Box
+            alignItems={AlignItems.center}
+            backgroundColor={TextColor.textDefault}
+            borderRadius={BorderRadius.full}
+            className="blockaid_banner__icon_wrapper"
+            display={Display.Flex}
+            justifyContent={JustifyContent.center}
+          >
+            <Icon
+              name={IconName.Check}
+              color={IconColor.primaryInverse}
+              size={IconSize.Sm}
+            />
+          </Box>
+          <Text color={TextColor.textDefault} marginLeft={2}>
+            {t('blockaidNoAlerts')}
+          </Text>
+        </Box>
+      );
+    }
```

Comment:
We do not want to show the banned if loading banner was not diaplyed.

Reply:
to get the final response, benign, when would it pass through loading vs not loading? Would the loading indicator not show if PPOM data has been loaded beforehand and thus there is an immediate response and no loading? 

IMO, the banner could be a handy indicator to the user that a validation has been completed. Else, I think it could be confusing to show "No alerts received" on the initial response gathering and then nothing on future requests. cc'ing design @SayaGT

Reply:
This is as per design and also how it works in mobile atm.